packages/ckeditor5-dev-env/lib/release-tools/tasks/update-package-versions.js

```diff
@@ -0,0 +1,232 @@
+		const parsedData = JSON.parse( currentFileData );
+
+		updateObjectProperty( parsedData, 'dependencies' );
+		updateObjectProperty( parsedData, 'devDependencies' );
+
+		const newFileData = JSON.stringify( parsedData, null, 2 ) + '\n';
+
+		if ( currentFileData !== newFileData ) {
+			updatedFiles++;
+
+			if ( dryRun ) {
+				differences.push( {
+					file,
+					content: diff( currentFileData, newFileData, { newlineIsToken: true } )
+				} );
+			} else {
+				fs.writeFileSync( file, newFileData, 'utf-8' );
+			}
+		}
+	}
+
+	return { found: packageJsonArray.length, updated: updatedFiles, differences };
+}
+
+/**
+ * This function takes an object and a property name, and modifies said property in that object. If any of the properties
+ * of that property matches the regex, its value will be updated to the value of `version` property from the root of the object.
+ *
+ * @param {Object} parsedPkgJson Object to update.
+ * @param {String} propertyName Name of the property to update.
+ */
+function updateObjectProperty( parsedPkgJson, propertyName ) {
+	// Update only the CKEditor 5 dependencies, except the *-dev and *-inspector.
+	const regex = /^@ckeditor\/ckeditor5-(?!dev|inspector)|^ckeditor5$/;
+	const version = parsedPkgJson.version;
+
+	for ( const dependency in parsedPkgJson[ propertyName ] ) {
+		if ( !regex.test( dependency ) ) {
+			continue;
+		}
+
+		parsedPkgJson[ propertyName ][ dependency ] = `^${ version }`;
+	}
+}
+
+/**
+ * Takes in raw changelog for a single file generated by jsdiff library, and returns formatted array of strings, containing
+ * human-readable, line by line changelog of a file, with removals and additions colored, as well as unnecessary chunks of
+ * unchanged file hidden.
+ *
+ * @param {Array<Object>} diff Array of changes for a single file generated by jsdiff library.
+ * @returns {Array<String>} Formatted changelog split into single lines.
+ */
+function formatDiff( diff ) {
+	const formattedDiff = [];
+	const regex = /(?<=":) (?=")/;
+
+	for ( let i = 0; i < diff.length; i++ ) {
+		const current = diff[ i ];
+		const next = diff[ i + 1 ];
+		const currentLines = current.value.split( '\n' );
+
+		const validReplaceSequence = current.removed && next.added && regex.test( current.value ) && regex.test( next.value );
+
+		if ( validReplaceSequence ) {
+			// Adding removals followed by additions (formatted).
+			formattedDiff.push( [
+				current.value.split( regex )[ 0 ],
+				' ',
+				chalk.red( current.value.split( regex )[ 1 ] ),
+				chalk.green( next.value.split( regex )[ 1 ] )
+			].join( '' ) );
+
+			i++;
+		} else if ( !current.added && !current.removed && currentLines.length > 8 ) {
+			// Cutting out the middle of a long streak of unchanged lines.
+			const shortenedLines = [
+				...currentLines.slice( 0, 3 ),
+				chalk.yellow( `[...${ currentLines.length - 7 } lines without changes...]` ),
+				...currentLines.slice( -4 )
+			].join( '\n' );
```

Comment:
Maybe it would be better to dim the unchanged lines? Yellow text draws attention.